composer.json

```
@@ -4,13 +4,13 @@
     "name": "flownative/neos-canto",
     "license": "MIT",
     "require": {
-        "php": "^7.4",
+        "php": "^8.0",
         "ext-json": "*",
-        "neos/flow": "^5.3 || ^6.0 || ^7.0 || dev-master",
-        "neos/media": "^4.3 || ^5.0 || ^7.0 || dev-master",
-        "guzzlehttp/guzzle": "6.*",
+        "neos/flow": "^8.0 || dev-master",
+        "neos/media": "^8.0 || dev-master",
```

I'd like this better, TBH:
```suggestion
        "neos/flow": "7.0 || ^8.0 || dev-master",
        "neos/media": "^7.0 || ^8.0 || dev-master",
```